docs/queries.rst

```diff
@@ -0,0 +1,232 @@
+Queries & ObjectTypes
+=====================
+
+Introduction
+------------
+
+The core functionality of GraphQL is to make queries in order to get data.
+
+Graphene-Django ships with a special ``DjangoObjectType`` that automagically transforms a Django Model
```

Comment:
As much as I love the word "automagically", I don't think we can use it in our docs. I'd omit it.